### PR TITLE
add a blog post about phoenix liveview

### PR DESCRIPTION
**Pull request by caironoleto**

### CODE REVIEW

File: content/posts/ciclo-de-vida-do-phoenix-liveview.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+[adaptei](https://github.com/caironoleto/phoenix-live-view-example)  para uma versão mais recente (dev!)
+do Phoenix, onde já podemos criar uma nova aplicação Phoenix incluindo o Phoenix LiveView
+(`mix phx.new path --live`).
+
+Nesse exemplo, é implementado um formulário com 3 passos e ao final "salva" no banco de dados. O código
+na view são vários `if/else`s e o controle do que é visto é feito na LiveView, usando a função `handle_event/3`
+para cada passo desse formulário.
+
+Aqui eu deixo o código do template para que você veja os vários `if/else`s (a organização disso entra em um
+segundo post!) e também o código da LiveView, com todas as funções que manipulam esse template.
```
Comment (cardosoana) [inline content/posts/ciclo-de-vida-do-phoenix-liveview.md:107]:
```suggestion
segundo post!) e também o código da LiveView com todas as funções que manipulam esse template.
```

File: content/posts/ciclo-de-vida-do-phoenix-liveview.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+title = "Ciclo de vida do Phoenix LiveView"
+date = "2020-04-01"
+author = "Cairo Noleto"
+cover = ""
+tags = ["elixir", "phoenix", "phoenix liveview", "myelixirstatus"]
+keywords = ["elixir", "phoenix", "phoenix liveview", "myelixirstatus"]
+showFullContent = false
++++
+
+Para quem não conhece, o [Phoenix LiveView](https://github.com/phoenixframework/phoenix_live_view) é uma biblioteca que funciona em cima do [Phoenix](https://www.phoenixframework.org/), framework web feito para [Elixir](https://elixir-lang.org/), e que trás todo o poder de se criar aplicações ricas em tempo real, tudo isso via server side e renderizando HTML.
```
Comment (LucasCFerraz) [inline content/posts/ciclo-de-vida-do-phoenix-liveview.md:11]:
```suggestion
Para quem não conhece, o [Phoenix LiveView](https://github.com/phoenixframework/phoenix_live_view) é uma biblioteca que funciona em cima do [Phoenix](https://www.phoenixframework.org/), framework web feito para [Elixir](https://elixir-lang.org/), e que traz todo o poder de criar aplicações ricas em tempo real, tudo isso via server side e renderizando HTML.
```

Comment (LucasCFerraz) [reply content/posts/ciclo-de-vida-do-phoenix-liveview.md:11]: The last sentence is confuse

File: content/posts/ciclo-de-vida-do-phoenix-liveview.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+
+A biblioteca usa todo o poder que Elixir trouxe de websockets e comunicação em tempo real para fazer com que sua página pareça uma aplicação sendo controlada pelo framework JavaScript da moda, mas que no final é só HTML + CSS e back-end.
```
Comment (LucasCFerraz) [inline content/posts/ciclo-de-vida-do-phoenix-liveview.md:13]:
```suggestion
A biblioteca usa todo o poder que o Elixir trouxe de websockets e comunicação em tempo real para fazer com que sua página pareça uma aplicação sendo controlada pelo framework JavaScript da moda, mas que no final é só HTML + CSS e back-end.
```

File: content/posts/ciclo-de-vida-do-phoenix-liveview.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+
+Antes de começarmos, eu espero que você já tenha seguido o [guia de instalação](https://hexdocs.pm/phoenix_live_view/installation.html) e que já tenha se familiarizado um pouco com LiveView. Recomendo ler os links antes de avançar.
```
Comment (LucasCFerraz) [inline content/posts/ciclo-de-vida-do-phoenix-liveview.md:15]:
```suggestion
Antes de começarmos, espero que você já tenha seguido o [guia de instalação](https://hexdocs.pm/phoenix_live_view/installation.html) e que já tenha se familiarizado um pouco com LiveView. Recomendo ler os links antes de avançar.
```

[the remark this answers is not in the document]
Comment (caironoleto) [reply content/posts/ciclo-de-vida-do-phoenix-liveview.md:15]: These links about phoenix live view and elixir before this line.

File: content/posts/ciclo-de-vida-do-phoenix-liveview.md (hunk 2 above)
Comment (caironoleto) [reply content/posts/ciclo-de-vida-do-phoenix-liveview.md:11]: Explain your confusion please :D

**Pull request merged**